tools/nxstyle.c

```diff
@@ -1265,27 +1265,38 @@ int main(int argc, char **argv, char **envp)
 
               if (have_upper && have_lower)
                 {
+                  /* REVISIT:  Although pre-processor definitions are
+                   * supposed to be all upper-case, there are exceptions
+                   * such as using 'p' for a decimal point or 'MHz'.
+                   * Those will be reported here, but probably should be
+                   * considered false alarms.
+                   */
+
+                  /* Ignore inttype.h strings beginning with PRIx */
+
+                  if (ident_index >= 4 &&
+                     (strncmp(&line[ident_index], "PRIx", 4) == 0))
+                    {
+                      /* No error */
+                    }
+
```

Comment:
We have some SD card constants and the like that are in the code base. Do they stay? If so adding an allowed words (grandfathered) table  may be of use here.

Reply:
This exception in nxstyle is because of standard definitionss that do not follow the coding style.
No nonstandard definitions should be grandfathered in; such definitions should eventually be changed to follow the coding standard.  Let's not formalize coding standard violations.  Let's fix them.